Kudu.Contracts/Settings/SettingsKeys.cs

```diff
@@ -36,5 +36,6 @@ public static class SettingsKeys
         public const string SiteExtensionsFeedUrl = "SCM_SITEEXTENSIONS_FEED_URL";
         public const string DisableDeploymentOnPush = "SCM_DISABLE_DEPLOY_ON_PUSH";
         public const string TouchWebConfigAfterDeployment = "SCM_TOUCH_WEBCONFIG_AFTER_DEPLOYMENT";
+        public const string MaxRandomDelayInSec = "SCM_MAX_RANDOM_START_DELAY_IN_SECONDS";
```

Comment:
Leave out the `_IN_SECONDS` part. Our other settings like `SCM_COMMAND_IDLE_TIMEOUT` and `SCM_LOGSTREAM_TIMEOUT` are all in seconds, and don't have that explicitly in their name.